cis_v200/docs/cis_v200_2_1_1.md

```diff
@@ -44,10 +44,10 @@ Using AWS Policy Generator:
 - `Principal` = *
 - `AWS Service` = Amazon S3
 - `Actions` = *
-- `Amazon Resource Name` = <ARN of the S3 Bucket>
+- `Amazon Resource Name` = &lt;ARN of the S3 Bucket&gt;
 
-5. Generate Policy.
-6. Copy the text and add it to the Bucket Policy.
+1. Generate Policy.
+2. Copy the text and add it to the Bucket Policy.
```

Comment:
@rajlearner17 Why have we changed the numbering here? Its a continuation from above.